Slate/ShellOperation.m

```diff
@@ -132,16 +132,19 @@ + (id)shellOperationFromString:(NSString *)shellOperation {
     }
   }
   NSString *commandAndArgs = [tokens lastObject];
+  NSString *c = [commandAndArgs stringByReplacingOccurrencesOfString:@"\\ " withString:SPACE_WORD];
```

Comment:
Would it be possible to find a better variable name to use here. Using `c` here is less expressive than I would care for the code to be.